data_generators/user_generators.py

```diff
@@ -8,60 +8,65 @@
 
 
 class Mentor:
+    '''Mentor schema'''
 
     def __init__(self):
         self.profile_id = generate_uuid(16)
+        self.mentor_intake_id = None
         self.email = "fake@email.com"
-        self.city = "Ashland"
-        self.state = "Oregon"
-        self.country = "USA"
-        self.first_name = random_first_name()
-        self.last_name = choice(last_names)
+        self.location = 'New York, New York'
+        self.in_US = True
+        self.name = f'{random_first_name()} {choice(last_names)}'
         self.current_comp = choice([
             "Boogle",
             "Amozonian",
             "Poptrist",
             "Macrohard",
             "Pineapple",
         ])
-        self.subject = choice(subjects)
+        self.tech_stack = choice(subjects)
         self.experience_level = choice(skill_levels)
-        self.job_help = self.subject == "Career Development"
+        self.job_help = self.tech_stack == "Career Development"
         self.industry_knowledge = percent_true(90)
         self.pair_programming = percent_true(90)
+        self.can_commit = True
+        self.how_commit = 'String'
         self.other_info = "Notes"
 
 
 class Mentee:
+    '''Mentee schema'''
+
     def __init__(self):
         self.profile_id = generate_uuid(16)
-        self.first_name = random_first_name()
-        self.last_name = choice(last_names)
+        self.mentee_intake_id = None
+        self.name = f'{random_first_name()} {choice(last_names)}'
         self.email = "fake@email.com"
-        self.city = "Ashland"
-        self.state = "Oregon"
-        self.country = "USA"
+        self.location = 'New York, New York'
+        self.in_US = True
         self.formerly_incarcerated = percent_true(80)
         self.underrepresented_group = percent_true(70)
         self.low_income = percent_true(70)
+        self.convictions = 'String of convictions'
         if self.formerly_incarcerated:
             self.list_convictions = sample(convictions, k=randint(1, 3))
         else:
             self.list_convictions = []
-        self.subject = choice(subjects)
+        self.tech_stack = choice(subjects)
         self.experience_level = choice(skill_levels)
-        self.job_help = self.subject == "Career Development"
+        self.job_help = self.tech_stack == "Career Development"
         self.industry_knowledge = percent_true(15)
         if self.job_help:
             self.pair_programming = False
         else:
             self.pair_programming = percent_true(60)
-        self.other_info = "Notes"
+        self.your_hope = 'String'
         self.need = choice(resource_items)
-        self.parole_restriction = choice(parole_restriction)
+        self.parole_restriction = ''
```

Comment:
line 65: I found out this should be a random bool... we have a function for that:
```
self.parole_restriction = percent_true(50)
```